client/evm_contract.go

```diff
@@ -0,0 +1,59 @@
+package client
+
+import (
+	"github.com/gogo/protobuf/proto"
+	"github.com/loomnetwork/go-loom"
+	"github.com/loomnetwork/go-loom/auth"
+	"github.com/loomnetwork/go-loom/vm"
+)
+
+// Contract provides a thin abstraction over DAppChainClient that makes it easier to perform
+// read & write operations on a contract running on a Loom DAppChain.
+type EvmContract struct {
+	client  *DAppChainRPCClient
+	Address loom.Address
+	Name    string
+}
+
+func NewEvmContract(client *DAppChainRPCClient, contractAddr loom.LocalAddress) *EvmContract {
```

Comment:
`NewEvmContract` -> `NewContract`

Reply:
This conflicts with the `NewContract` in contract.go. Unless evm_contract is in a separate package.